packages/server/tsconfig.build.json

```diff
@@ -10,7 +10,15 @@
     "incremental": true,
     "types": ["node", "jest"],
     "outDir": "dist/src",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": ".",
+    "paths": {
```

Comment:
What's the implication of paths being moved into the build ts config? Do we use the `-p` now somewhere?